src/Makefile

```diff
@@ -1,5 +1,12 @@
 debug ?= yes
 static ?= no
+GZIP_MAN ?= yes
```

Comment:
Is that a conventional name ? if not, can it use the same naming conventions as the rest, namely lowercase ?

Reply:
It's not a conventional name, I'll lowercase it.